### ISSUE TITLE
PrepareClone not working

### ISSUE THREAD

**Issue by sameer-kumar** [MicrosoftDocs/vsts-docs#301]
I'm trying to follow this article for my TFS 2017.1 and when I reached section - **[(Clone option) Reconfigure server IDs and remap databases](https://docs.microsoft.com/en-us/vsts/tfs-server/admin/move-clone-hardware#clone-option-reconfigure-server-ids-and-remap-databases)**, it gives me an error stating that "_Unrecognized command: PrepareClone_". 
I am trying to run this command:

`TFSConfig PrepareClone /SQLInstance:MySandbox1 /DatabaseName:Tfs_Configuration`

Any help is appreciated.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f39ca7f9-03a8-98ed-901d-630eadb2d4fc
* Version Independent ID: cc5b1311-9e38-17f2-9709-f5725313293c
* Content: [Move Team Foundation Server from one hardware configuration to another | Microsoft Docs](https://docs.microsoft.com/en-us/vsts/tfs-server/admin/move-clone-hardware#clone-option-reconfigure-server-ids-and-remap-databases)
* Content Source: [docs/tfs-server/admin/move-clone-hardware.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/tfs-server/admin/move-clone-hardware.md)
* Service: **unspecified**
* GitHub Login: @douge
* Microsoft Alias: **elbatk**

**Comment by elbatk** [comment]
Hi @sameer-kumar , thanks for the question - I'm going to forward this to the product team and try to get an answer for you shortly!

**Comment by pabrams** [comment]
On my TFS 2017, the PrepareClone command doesn't work, and when I try to select a more recent version of TFS than is shown on this page, it tells me it doesn't exist.

**Comment by elbatk** [comment]
Hey @sameer-kumar and @pabrams , I got this response from the product team:

PrepareClone used to be recommended for use before standing up a new TFS deployment using a database backup already in production on another server. The idea was to disable things like scheduled backups that would otherwise compete with production for an external resource. (In other words, scheduled backups copy backups to a file share. If a cloned server kept that configuration in place, it would attempt to write backups to the same place production is already writing them…)

This command is no longer required, since we incorporated its functionality into the Pre-Production Upgrade and Clone scenarios in the configuration wizard. 

**Should I add this to the docs?**
